components/navigation/NavBar.js

```diff
@@ -138,3 +144,8 @@ export default function NavBar({
     </div>
   )
 }
+
+NavBar.propTypes = {
+  className: '',
+  hideLogo: false,
+}
```

Comment:
Why did you add proptypes of Navbar here?

Reply:
Removed